opensrp-app/src/main/java/org/smartregister/repository/AllSharedPreferences.java

```diff
@@ -278,5 +280,29 @@ public void updateLastSettingsSyncTimeStamp(long lastSettingsSync) {
     public long fetchLastSettingsSyncTimeStamp() {
         return preferences.getLong(LAST_SETTINGS_SYNC_TIMESTAMP, 0);
     }
+
+    public boolean isMigratedToSqlite4() {
+        return preferences.getBoolean(MIGRATED_TO_SQLITE_4, false);
+    }
+
+    public void setMigratedToSqlite4() {
+        preferences.edit().putBoolean(MIGRATED_TO_SQLITE_4, true).commit();
+    }
+
+    public int getLastPeerToPeerSyncProcessedEvent() {
```

Comment:
Since this is only used by `isPeerToPeerUnprocessedEvents` and the value is not used, convert this to a boolean preference

Reply:
It's used inside the the `P2pProcessRecordsService` to figure out from which event we should continue processing in case the  process was halted or died for one reason or another